File: Day-16/Pattern26.py
n = int(input())
def PatternTwentySix(n):
    i = 1
    printNum = 1
    while (i <=n):
        space = n - i
        while (space):
            print(' ', end='')
            space -= 1
        
        star = 1
        while(star <= i):
            print(printNum,end='')
            star +=1
            printNum +=1 
        
        i+=1
        print(end='\n')       
PatternTwentySix(n)

File: Day-17/Pattern26.js
function PatternTwentySix(n) {
  let i = 1;
  let printNum = 1;
  while (i <= n) {
    let str = "";
    let spaces = n - i;
    while (spaces) {
      str += " ";
      spaces--;
    }

    let nums = i;
    while (nums) {
      str += printNum;
      printNum++;
      nums--;
    }

    console.log(str);
    i++;
  }
}

PatternTwentySix(4);
